src/web/routes/admin/teams/types.ts

```diff
@@ -23,8 +24,9 @@ export type ConnectFigmaTeamRequest = Request<
 >;
 
 export type ConnectTeamResponseBody = {
-	readonly success: string[];
```

Comment:
**note:** Interestingly, the app have never returned the initially defined `ConnectTeamResponseBody`. Updated the implementation to return what Roy wanted to return here: https://github.com/atlassian-labs/figma-for-jira/pull/118/files#diff-473697600a010a6f00bb27d2fa9a0c1d0bcd7bfa25071d3b203318cb8c6d7c2fR43